index.js

```diff
@@ -42,6 +48,7 @@ module.exports = {
         },
         redisDeployClient: function(context) {
           var redisOptions = this.pluginConfig;
+          redisOptions.port = this.readConfig('port');
```

Comment:
@lukemelia How do you feel about this modifying the config object? Feels like this is a perfect candidate for need to have passed in a clone of the config so that any modifications to it don't affect anything outside this hook.